corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -86,16 +86,14 @@ def __init__(self, attrs=None, choices=(), ko_value=None):
         self.choices = list(choices)
 
     def render(self, name, value, attrs=None, choices=()):
-        value = '' if value is None else value
+        self.value = '' if value is None else value
         final_attrs = self.build_attrs(attrs, name=name)
 
         return format_html(
             '<input{0} type="text" data-bind="select2: {1}, {2}">',
             flatatt(final_attrs),
             json.dumps(self._choices_for_binding(choices)),
-            'value: {}'.format(
-                self.ko_value
-            ) if self.ko_value else ""
+            'value: {}'.format(self.ko_value or self.value)
```

Comment:
Right, but when `ko_value` isn't provided, there is no knockout variable to bind to.  This widget is called both in contexts where it's bound to a model, and contexts where it isn't.  This falls back to the old behavior when it's not a knockout-bound widget.  That is, https://github.com/dimagi/commcare-hq/pull/16845/files#diff-66cf616ff719298fed0e5e58ca5165ccL123
Actually I just realized that this should be
```python
'value: {}'.format(self.ko_value) if self.ko_value else 'value: "{}"'.format(self.value) 
```
(I forgot the quotes when it's a literal)  Does that sound right to you?